cmd/containerd-shim-runc-v2/process/exec.go

```diff
@@ -204,6 +204,9 @@ func (e *execProcess) start(ctx context.Context) (err error) {
 	if socket != nil {
 		opts.ConsoleSocket = socket
 	}
+	// generate a special log file for every exec process.
+	e.parent.runtime.Log = filepath.Join(filepath.Dir(e.parent.runtime.Log), fmt.Sprintf("%s-exec.log", e.id))
+	defer os.RemoveAll(e.parent.runtime.Log)
```

Comment:
at the moment this is not a directory.. it's just a single log file or did I miss some log rotation logic?